redaxo/src/core/lib/console/sync_assets.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ * 
+ * @auth staabm
+ *
+ * @internal
+ */
+class rex_command_sync_assets extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Sync files of /assets with addons/assets core/assets folders');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output);
+
+        foreach(rex_package::getInstalledPackages() as $package) {
+            if ($package instanceof rex_addon) {
+                $feAssetsPath = rex_path::addonAssets($package->getName());
+                $beAssetsPath = rex_path::addon($package->getName(), 'assets/');
+            } else {
+                // we assume "plugin" assets folder is a subfolder of the "addon" assets folder
+                // and will therefore already be synced when handling the addon
+                continue;
+                // $feAssetsPath = rex_path::pluginAssets($package->getAddon()->getName(), $package->getName());
+                // $beAssetsPath = rex_path::plugin($package->getAddon()->getName(), $package->getName(), 'assets/');
+            }
+
+            // dont create top level "assets" folder when it doesnt exist
+            if (!file_exists($feAssetsPath)) {
+                $io->comment("skip not existing assets/ folder: $feAssetsPath");

```

Comment:
Via verbosity steuern

Reply:
Ich glaube, ich würde die Ordner durchaus anlegen, wenn noch nicht vorhanden. In beide Richtungen.
Nützlich beim initialen Entwickeln eines Addons zum Beispiel. Der eine legt seine ersten Assets vielleicht im fe-Pfad an, der andere im src-Ordner. Für beide wäre der Command dann nützlich, um sie auch initial an die andere Stelle zu bekommen.